js/main.js

```diff
@@ -1,10 +1,11 @@
 $("#more").click(function(){
     $("section").css({"position": "fixed"});
-    $("#info").animate({"left": "10%",}, 200);
-    $("#info").animate({"left": "0%",}, 200);
+    $("#info").animate({"left": "0%"}, 200);
```

Comment:
these are JQuery, i kept it that way because not all browsers have CSS3 translate support :P